lib/src/hmssdk.dart

```diff
@@ -219,12 +219,16 @@ class HMSSDK {
   /// The [hmsActionResultListener] informs about whether the message was successfully sent, or the kind of error if not.
   void sendGroupMessage(
       {required String message,
-      required String roleName,
+      required List<HMSRole> roles,
       String? type,
       HMSActionResultListener? hmsActionResultListener}) async {
-    var arugments = {"message": message, "type": type, "role_name": roleName};
+
+    List<String> rolesMap = [];
+    roles.forEach((role)=> rolesMap.add(role.name));
+
+    var arugments = {"message": message, "type": type, "roles": rolesMap};
```

Comment:
typo error: should be `arguments`
Also, `arguments` passed back to the listener would have HMSRole objects